src/gc.rs

```diff
@@ -39,6 +39,9 @@ pub struct Gc<T: ?Sized> {
     _phantom: PhantomData<T>,
 }
 
+impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<Gc<U>> for Gc<T> {}
+impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<Gc<U>> for Gc<T> {}
```

Comment:
Is this just a marker trait that says "It's ok to dispatch methods on me when used as a trait object"?

Reply:
Pretty much. https://doc.rust-lang.org/std/ops/trait.DispatchFromDyn.html